### PR TITLE
 default keymap for dactyl_manuform 5x6

### PR DESCRIPTION
**Pull request by zwnk**
default keymap added to prevent travis build errors.

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/default/config.h]: And `#pragma once` here.

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#define USE_SERIAL
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+//#define EE_HANDS
+// Rows are doubled-up
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 6
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/default/config.h]: Remove this line.

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,75 @@
+#include "dactyl_manuform.h"
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/default/keymap.c]: Replace all of these includes with just `#include QMK_KEYBOARD_H`

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                                               _______,_______,            KC_EQL ,_______,
+                                               _______,_______,            _______,_______,
+                                               _______,_______,            _______,_______,
+                                               _______,_______,            _______,_______
+  ),
+
+  };
+
+
+void persistant_default_layer_set(uint16_t default_layer) {
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/default/keymap.c]: Remove this function (it's out of date, and not needed)

**Comment by zwnk** [comment]
```
> Compiling: keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.c                       <command-line>:0:16: error: empty filename in #include
keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.c:3:8: error: unknown type name ‘keymap_config_t’
 extern keymap_config_t keymap_config;
        ^
keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.c:11:7: error: unknown type name ‘uint16_t’
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
       ^
keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.c:11:24: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘keymaps’
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
                        ^
 [ERRORS]
 |
 |
 |
tmk_core/rules.mk:357: recipe for target '.build/obj_handwired_dactyl_manuform_5x6_impstyle/keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.o' failed
make[1]: *** [.build/obj_handwired_dactyl_manuform_5x6_impstyle/keyboards/handwired/dactyl_manuform/5x6/keymaps/impstyle/keymap.o] Error 1
Makefile:529: recipe for target 'handwired/dactyl_manuform/5x6:impstyle' failed
```

is the error i get after changing it.

**Comment by drashna** [comment]
Ah, this should be renamed: https://github.com/qmk/qmk_firmware/blob/2ed29697e8b0ec6c6f2d7dc5b592079ba6b65346/keyboards/handwired/dactyl_manuform/5x6/dactyl_manuform.c

Both the c and h should be "5x6"

**Comment by drashna** [comment]
@zwnk I made a pull request to your fork.  I checked, it compiles, and works.  You should be able to merge that, and then that will update your PR, and should get things working.

**Comment by zwnk** [comment]
Thank you drashna, you were faster replying then me searching the docu.
now i get
```
Compiling: keyboards/handwired/dactyl_manuform/5x6/5x6.c                                            [OK]
Compiling: tmk_core/common/command.c                                                               tmk_core/common/command.c: In function ‘command_proc’:
tmk_core/common/command.c:80:13: error: implicit declaration of function ‘IS_COMMAND’ [-Werror=implicit-function-declaration]
             if (!IS_COMMAND())
             ^
cc1: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
tmk_core/rules.mk:357: recipe for target '.build/obj_handwired_dactyl_manuform_5x6_impstyle/common/command.o' failed
make[1]: *** [.build/obj_handwired_dactyl_manuform_5x6_impstyle/common/command.o] Error 1
Makefile:529: recipe for target 'handwired/dactyl_manuform/5x6:impstyle' failed
```

**Comment by drashna** [comment]
That indicates that you have the IS_COMMAND thing defined twice.  Check your config.h.   Also, make sure that your config.h doesn't include any other config.h

**Comment by drashna** [comment]
Travis passes the build, so I'm merging this.  (sorry for missing the issues the first time around!)

**Pull request merged**